src/plugins/intel_npu/src/plugin/npuw/llm_infer_request.cpp

```diff
@@ -29,10 +29,8 @@ ov::SoPtr<ov::ITensor> make_tensor_slice(ov::SoPtr<ov::ITensor> tensor,
 }
 }  // anonymous namespace
 
-ov::npuw::LLMInferRequest::LLMInferRequest(const std::shared_ptr<ov::npuw::LLMCompiledModel>& compiled_model,
-                                           const ov::npuw::LLMCompiledModel::KVCacheDesc& kvcache_desc)
-    : ov::ISyncInferRequest(compiled_model),
-      m_kvcache_desc(kvcache_desc) {
+ov::npuw::LLMInferRequest::LLMInferRequest(const std::shared_ptr<ov::npuw::LLMCompiledModel>& compiled_model)
+    : ov::ISyncInferRequest(compiled_model) {
     m_kvcache_request = compiled_model->m_kvcache_compiled->create_infer_request();
     m_prefill_request = compiled_model->m_prefill_compiled->create_infer_request();
```

Comment:
If we keep `npuw_llm_compiled_model` anyway, I believe we don't need additional variables for `m_prefill_request` and `m_kvcache_request`, do we?

Reply:
They are created right in this class, so I suppose we can keep them here

Reply:
Oh, it's about `requests` then OK